README.md

```diff
@@ -6,195 +6,252 @@
   <img width="400" alt="Example's react-content-loader" src="https://user-images.githubusercontent.com/4838076/34308760-ec55df82-e735-11e7-843b-2e311fa7b7d0.gif" />
 </p>
 
-React component that uses SVG to create a collection of loaders which simulates the structure of the
-content that will be loaded, similar to Facebook cards loaders.
-
-[![Size](https://img.shields.io/badge/gzip%20size-1.4kb-brightgreen.svg)]()
-[![Dependencies](https://img.shields.io/badge/dependencies-0-brightgreen.svg)]()
-[![CircleCI](https://img.shields.io/circleci/project/github/RedSparr0w/node-csgo-parser.svg)]()
-[![Coverage](https://img.shields.io/badge/coverage-100%25-brightgreen.svg)]()
-[![](https://data.jsdelivr.com/v1/package/npm/react-content-loader/badge)](https://www.jsdelivr.com/package/npm/react-content-loader)
+SVG-Powered component to easily create placeholder loadings, like Facebook cards loading.
```

Comment:
SVG-Powered component to easily create placeholder loadings (like Facebook's cards loading).